src/sortStudents.ts

```diff
@@ -1,16 +1,58 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAverageGrade(grades: number[]): number {
+  return grades.length
+    ? grades.reduce((sum, grade) => sum + grade, 0) / grades.length
+    : 0;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copyStudents: Student[] = [...students];
```

Comment:
sounds like a name for a function, as functions start with a verb. But if we do it vice versa - it becomes clear we're dealing with a variable
```suggestion
  const studentsCopy: Student[] = [...students];
```